h/static/scripts/annotator/monkey.coffee

```diff
@@ -9,6 +9,12 @@ Util = Annotator.Util
 delete Annotator.prototype.events[".annotator-hl mouseover"]
 delete Annotator.prototype.events[".annotator-hl mouseout"]
 
+# Add createHighlight
+Annotator.prototype.createHighlight = ->
+  annotation = {$highlight: true}
```

Comment:
Ouch. Aren't we fighting *against* our internal data being stored inside annotations? This seems to go into the opposite direction.